openshift_metrics/invoice.py

```diff
@@ -9,20 +9,23 @@
 GPU_A100 = "NVIDIA-A100-40GB"
 GPU_A100_SXM4 = "NVIDIA-A100-SXM4-40GB"
 GPU_V100 = "Tesla-V100-PCIE-32GB"
+GPU_H100 = "???-H100-???-???" # TODO Confirm name of H100 in Openstack
 GPU_UNKNOWN_TYPE = "GPU_UNKNOWN_TYPE"
 
 # GPU Resource - MIG Geometries
 # A100 Strategies
 MIG_1G_5GB = "nvidia.com/mig-1g.5gb"
 MIG_2G_10GB = "nvidia.com/mig-2g.10gb"
 MIG_3G_20GB = "nvidia.com/mig-3g.20gb"
+# TODO Will we allow MIG on H100s?
```

Comment:
@joachimweyl @naved001 @knikolla Will the H100 allow MIG?

Reply:
@QuanMPhm The support for MIG billing can be added later once it becomes clear, so for this PR leave it out of it.